tutorials/forest-cover-loss-estimation/index.md

```diff
@@ -0,0 +1,173 @@
+
+## Context
+
+The Food and Agriculture Organization of the United Nations (FAO) defines forests as "Land spanning more than 0.5 hectares with trees higher than 5 meters and a canopy cover of more than 10 percent, or trees able to reach these thresholds in situ. It does not include land that is predominantly under agricultural or urban land use" (FAO 2015). 
+
+In various reporting of forest and/or land-use related data, countries may have different forest definitions, including these parameters, forest types or land use categories. 
+
+## Instructions
+### Select a country and set parameters
+
+This tutorial selected Bolivia as an example, with the minimum canopy cover of 10% and minimum forest area of 0.5 ha (the national definition may be different). 
+```js
+var country = 'Bolivia'; // selected country (e.g. Bolivia)
+var cc = ee.Number(10); // canopy cover percentage (e.g. 10%)
+var pixels = ee.Number(6); // minimum forest area in pixels (e.g. 6 pixels, approximately 0.5 ha in this example)
+var lossPixels = ee.Number(6); // minimum mapping area for tree loss (usually same as the minimum forest area)
+
+// Load country features from Large Scale International Boundary (LSIB) dataset.
+var countries = ee.FeatureCollection('USDOS/LSIB_SIMPLE/2017');
+var selected = countries.filter(ee.Filter.eq('country_na', ee.String(country)));
+```
+
+Depending on the location of the country, the number of pixels that make up for 0.5 ha would differ. You can adjust this by calculating the actual minimum areas you were using (see step 6 below).
+
+### Tree cover
+
+Currently, Google Earth Engine has several tree cover datasets in the catalogue, including the Global Forest Change (GFC) (year 2000) and GLCF: Landsat Tree Cover Continuous Fields (2000, 2005, and 2010). Here, we use the Global Forest Change dataset. 
+```js
+var gfc2018 = ee.Image('UMD/hansen/global_forest_change_2018_v1_6');
+```
+
+1. Select 'treecover2000' in the Global Forest Change dataset
+```js
+var canopyCover = gfc2018.select(['treecover2000']);
+```
+2. Apply the minimum canopy cover percentage (e.g. greater than or equal to 10%). Use `selfMask()` to mask value zero
+```js
+var canopyCover10 = canopyCover.gte(cc).selfMask();
+```
+3. Apply the minimum area requirement using `connectedPixelCount` (e.g. greater than or equal to 6 pixels). Note that if no kernel is passed to `connectedPixelCount`, 8 neighbor adjacency is used to determine connectivity
+```js
+// Use connectedPixelCount() to get contiguous area.
+var contArea = canopyCover10.connectedPixelCount();
+// Apply the minimum area requirement. 
+var minArea = contArea.gte(pixels).selfMask();
+```
+4. Scale the results in nominal value based on to the dataset's projection to display on the map
```

Comment:
Could you also clarify briefly why the reprojection and change from continuous to nominal is needed?

Reply:
Done